index.html

```diff
@@ -244,17 +244,18 @@ <h3>Conceptual Framework</h3>
 data format since, in the average case, large numbers of credentials will
 remain unrevoked. This will ensure long sections of bits that are the same
 value and thus highly compressible using run-length compression techniques
-such as GZIP [[RFC1952]]. The default bitstring size is 16KB (131,072 entries),
-and when only a handful of <a>verifiable credentials</a> are revoked, the
-compressed bitstring size is reduced down to a few hundred bytes.
+such as GZIP [[RFC1952]]. The default status list size is 131,072 entries,
+equivalent to 16&nbsp;KB of single bit values. When only a handful of
+<a>verifiable credentials</a> are revoked, GZIP compresses the bitstring
+to a few hundred bytes.
         </p>
 
         <p>
 Another benefit of using a bitstring is that it enables large numbers of
-<a>verifiable credential</a> statuses to be placed in the same  list.
-This specification utilizes a minimum bitstring length of 131,072 (16KB). This
-population size ensures an adequate amount of herd  privacy in the average case.
-If better herd privacy is required, the bitstring can be made to be larger.
+<a>verifiable credential</a> statuses to be placed in the same list.
+This specification uses a minimum list length of 131,072. This
+size ensures an adequate amount of herd privacy in the average case.
```

Comment:
The suggestion has been to replace "herd" with "group" everywhere as friendlier terminology.

```suggestion
size ensures an adequate amount of group privacy in the average case.
```

Reply:
Yes, I was thinking the same thing while processing these PRs. I was going to raise an entirely separate PR to do the "herd" -> "group" renaming.

Reply:
Change raised in PR #121.